composer.json

```diff
@@ -4,14 +4,15 @@
 	"license": "GPL-2.0-or-later",
 	"require": {
 		"php": ">=8.1",
-		"doctrine/orm": "^2.11",
-		"doctrine/dbal": "^3.3",
+		"doctrine/orm": "~2.18 | ~3.0",
```

Comment:
How come we accept different versions here?

If it's to please Travis we should be able to use the more recent ones

Reply:
We can ask Gabriel about this when he is back.